.github/boring-cyborg.yml

```diff
@@ -27,8 +27,8 @@ labelPRBasedOnFilePath:
     - providers/tests/alibaba/**/*
     - providers/tests/system/alibaba/**/*
 
-  provider:amazon-aws:
-    - providers/src/airflow/providers/amazon/aws/**/*
```

Comment:
Why?
Ar some point there can be amazon provider that is not aws (integration with sellers API for Amazon marketplaces)

Reply:
Becasue it's inconsistent with all the other providers. We hae  `apache-airflow-providers-amazon`  not `apache-airlfow-providers-amazon-aws`